examples/ChatQnA/deploy/index.rst

```diff
@@ -19,9 +19,10 @@ Single Node
 Kubernetes
 **********
 
+* Getting Started <k8s_getting_started.md>
```

Comment:
Not quite right.  We need to get these two new docs into the toctree, so change this section under the heading to look like this:
```

.. toctree::
   :maxdepth: 1

   k8s_getting_started
   TGI on Xeon with Helm Charts <k8s_helm>

* Xeon & Gaudi with GMC
* Xeon & Gaudi without GMC
```